# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pyzor/makefile.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2003 Clifford Wolf
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -urN pyzor-0.4.0.orig/Makefile pyzor-0.4.0/Makefile
--- pyzor-0.4.0.orig/Makefile	1969-12-31 19:00:00.000000000 -0500
+++ pyzor-0.4.0/Makefile	2003-06-25 13:29:43.000000000 -0400
@@ -0,0 +1,4 @@
+all:
+	python setup.py build
+install:
+	python setup.py install
